src/modules/commander/PreflightCheck.cpp

```diff
@@ -223,7 +223,7 @@ static bool magConsistencyCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s
 
 	if (sensors.mag_inconsistency_ga > test_limit) {
 		if (report_status) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAG SENSORS INCONSISTENT");
+			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Compass Sensors inconsistent");
```

Comment:
Compass Sensors?

Reply:
From a user perspective I think "Compass Sensors inconsistent" is more easily understood and more consistent with the user interface in QGC.  However, "Mag" and "Mag Sensor(s)" is still used in some of the other warnings.

On a side note: I think it is still okay if "MAG" is used in the parameters and other developer and integrator focused areas.  In user focused areas (UX and non-developer documentation) "compass" is more easily understood.

Reply:
Compass instead of Mag seems better to me as long as we're consistent everywhere. Compass **Sensors** is what I was questioning that seemed inconsistent with usage.

Reply:
So which is better: "Preflight Fail: Compass inconsistent" or "Preflight Fail: Compasses inconsistent"?